third_party/jemalloc.BUILD

```diff
@@ -183,12 +190,12 @@ sh_binary(
     srcs = ["include/jemalloc/internal/size_classes.sh"],
 )
 
-# Size classes for Linux x86_64. Update if adding builds for other
+# Size classes for Linux x86_64 and ppc64le. Update if adding builds for other
 # architectures. See size_classes.sh for details on the arguments.
 genrule(
     name = "size_classes_h",
     outs = ["include/jemalloc/internal/size_classes.h"],
-    cmd = "$(location :size_classes_sh) \"3 4\" 3 12 2 >$@",
+    cmd = "if [ $$(uname -p) = ppc64le ]; then $(location :size_classes_sh) \"3 4\" 3 16 2 >$@; else $(location :size_classes_sh) \"3 4\" 3 12 2 >$@; fi",
```

Comment:
I tried avoiding uname here as it won't work on windows. Tried to use $(TARGET_CPU) which gives correct value on command line "ppc" but not when this file is executed. That time, it takes "local" as the cpu value defined in crosstool.tpl file. I'll update this code if anyone has any better suggestions other than using uname to detect architecture.